internal/cluster/node.go

```diff
@@ -78,6 +85,10 @@ func (n *NodeInfo) populateNodeIpList() error {
 		return fmt.Errorf("can't fetch node list: %+v ", err)
 	}
 
+	if len(nodeList.Items) == 0 {
+		return fmt.Errorf("can't fetch node list: %+v ", err)
+	}
+
```

Comment:
if none of the nodes have externalIps available, should we consider internalIp too? This would make it easier for on-prem and kind installations. For eg. in k3s installation, it shows node public ip as `internalIP`

This is from my kind node:

```
    addresses:
    - address: 172.19.0.2
      type: InternalIP
    - address: kind-control-plane
      type: Hostname
```
Here the IP is docker network ip, which is reachable from other kind clusters. It should be considered as NodeIP even through it is configured as `InternalIP` here.

What do you think?

Reply:
sure, I will include this

Reply:
Will do this in separate story